SenlaTask4/SenlaTask4/MainViewController.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  MainViewController.swift
+//  SenlaTask4
+//
+//  Created by Никита Макаревич on 31.03.2022.
+//
+
+import Foundation
+import UIKit
+
+protocol MainVCDelegate : AnyObject{
+    func changeRules(gameMode : Bool)
+    func changeLanguage(lanEn : Bool)
+    var rulesWithoutRepeat : Bool {get set}
+    var languageIsEnglish : Bool {get set}
+}
+
+class MainViewController : UIViewController, MainVCDelegate {
+    
+    var languageIsEnglish: Bool = false
+    var rulesWithoutRepeat : Bool = false
+
+    let options = ["Камень", "Ножницы", "Бумага"]
```

Comment:
Проперти нужно прихватить если они относятся только к конкретному экрану

Reply:
Проперти нужно приватить если они относятся только к конкретному экрану
```suggestion
    private let options = ["Камень", "Ножницы", "Бумага"]
```